example/lib/main.dart

```diff
@@ -48,38 +87,213 @@ class _MyAppState extends State<MyApp> {
+                    ],
+                  ),
+                  Row(
+                    mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+                    children: [
+                      ElevatedButton(
+                        onPressed: () {
+                          getLocation();
+                        },
+                        child: Text(
+                          StringConstants.locationButton,
+                          style: TextStyle(
+                            fontSize: NumericConstants.fontSizeLocationsText,
+                            color: Colors.grey.shade300,
+                          ),
+                        ),
+                        style: ElevatedButton.styleFrom(
+                          primary: Colors.red.shade900,
+                        ),
+                      ),
+                      ElevatedButton(
+                        onPressed: () {
+                          stopLocation();
+                        },
+                        child: Text(
+                          StringConstants.stopLocationButton,
+                          style: TextStyle(
+                            fontSize: NumericConstants.fontSizeLocationsText,
+                            color: Colors.grey.shade300,
+                          ),
+                        ),
+                        style: ElevatedButton.styleFrom(
+                          primary: Colors.red.shade900,
+                        ),
+                      ),
+                    ],
+                  ),
+                  StreamBuilder(
+                    stream: AndroidPlugin.locationStream,
+                    builder: (context, snapshot) {
+                      return snapshot.hasData
+                          ? Container(
+                              padding: new EdgeInsets.all(
+                                NumericConstants.coordinatesPadding,
+                              ),
+                              decoration: BoxDecoration(
+                                border: Border.all(
+                                  color: Colors.black,
+                                ),
+                              ),
+                              child: Column(
+                                children: [
+                                  Text(
+                                    snapshot.data.toString().split("%").first,
+                                    style: TextStyle(
+                                      fontSize: NumericConstants
+                                          .coordinatesTextFontSize,
+                                    ),
+                                  ),
+                                  Text(
+                                    snapshot.data.toString().split("%").last,
+                                    style: TextStyle(
+                                      fontSize: NumericConstants
+                                          .coordinatesTextFontSize,
+                                    ),
```

Comment:
When you pressed on stop location it's shown two times the same message, can you try yo show only one 'Location Request Stoped'?

Reply:
Fixed